pom.xml

```diff
@@ -446,6 +446,33 @@
 					<autoVersionSubmodules>true</autoVersionSubmodules>
 				</configuration>
 			</plugin>
+      
```

Comment:
@bryan4887 We removed this from here a while ago because building source and javadocs during a build takes such a long time, given the vast amount of modules in this project. Do you absolutely need this?